modules/data.atmosphere/R/met.process.R

```diff
@@ -10,21 +10,23 @@
 ##' @param host Host info from settings file
 ##' @param dbparms  database settings from settings file
 ##' @param dir  directory to write outputs to
+##' @param overwrite Whether to force met.process to proceed, even if check criteria don't think it's necessary. Will overwrite any resulting files that happen to exist already. 
 ##'
-##' @author Elizabeth Cowdery, Michael Dietze, Ankur Desai, James Simkins
-met.process <- function(site, input_met, start_date, end_date, model, host, dbparms, dir, browndog=NULL){
-# browser()
+##' @author Elizabeth Cowdery, Michael Dietze, Ankur Desai, James Simkins, Ryan Kelly
+met.process <- function(
```

Comment:
Most of what's changed here is formatting and the above mentioned move of "modules" into their own functions. No real functionality changes, IIRC.